utils/tf_solver.py

```diff
@@ -112,12 +115,21 @@ def set_objective(
         self.image_width = image_width
 
         if self.data_aug:
-            data_aug_list = [
-                tf.keras.layers.ZeroPadding2D(padding=4),
-                tf.keras.layers.RandomCrop(
+            if self.image_width < 128:
+                crop = tf.keras.layers.RandomCrop(
                     height=self.image_width,
                     width=self.image_width,
-                ),
+                )
+            else:
+                # random resize crop equivalent in tf
+                crop = RandomResizedCrop(
+                    scale=(0.08, 1.0),
+                    ratio=(0.75, 1.33),
+                    crop_shape=(self.image_width, self.image_width),
+                )
+            data_aug_list = [
+                tf.keras.layers.ZeroPadding2D(padding=4),
```

Comment:
Define this above the `if` and then append the correct crop. This avoids creating an extra local var just to append it in the sequence and keep a chronological definition.